app/components/verkiezingen/draft-mandataris-list.js

```diff
@@ -60,7 +56,7 @@ export default class DraftMandatarisListComponent extends Component {
       queryParams['filter']['is-bestuurlijke-alias-van'] = this.args.filter;
     }
 
-    this.mandatarissen = this.store.query('mandataris', queryParams);
+    this.mandatarissen = await this.store.query('mandataris', queryParams);
```

Comment:
We should await this haha else our sortedMandataris getter on L24 is throwiung an error that a promise is not iterable ;p